lib/handlebars/helpers/each.js

```diff
@@ -30,15 +25,11 @@ export default function(instance) {
         data.index = index;
         data.first = index === 0;
         data.last = !!last;
-
-        if (contextPath) {
-          data.contextPath = contextPath + field;
-        }
       }
 
       ret = ret + fn(context[field], {
         data: data,
-        blockParams: blockParams([context[field], field], [contextPath + field, null])
+        blockParams: blockParams([context[field], field], [field, null])
```

Comment:
I think the `blockParams` utility method was basically in support of `trackIds` can we remove completely?